extensions/wikia/AchievementsII/services/AchImageUploadService.class.php

```diff
@@ -23,71 +23,71 @@ private static function imagecopymerge_alpha($dst_im, $src_im, $dst_x, $dst_y, $
+				if( $alpha < $minalpha ) {
 					$minalpha = $alpha;
 				}
 			}
 		//loop through image pixels and modify alpha for each
-		for( $x = 0; $x < $w; $x++ ){
-			for( $y = 0; $y < $h; $y++ ){
+		for( $x = 0; $x < $w; $x++ ) {
+			for( $y = 0; $y < $h; $y++ ) {
 				//get current alpha value (represents the TANSPARENCY!)
 				$colorxy = imagecolorat( $src_im, $x, $y );
 				$alpha = ( $colorxy >> 24 ) & 0xFF;
 				//calculate new alpha
-				if( $minalpha !== 127 ){
+				if( $minalpha !== 127 ) {
 					$alpha = 127 + 127 * $pct * ( $alpha - 127 ) / ( 127 - $minalpha );
 				} else {
 					$alpha += 127 * $pct;
 				}
 				//get the color index with new alpha
 				$alphacolorxy = imagecolorallocatealpha( $src_im, ( $colorxy >> 16 ) & 0xFF, ( $colorxy >> 8 ) & 0xFF, $colorxy & 0xFF, $alpha );
 				//set pixel with the new color + opacity
-				if( !imagesetpixel( $src_im, $x, $y, $alphacolorxy ) ){
+				if( !imagesetpixel( $src_im, $x, $y, $alphacolorxy ) ) {
 					return false;
 				}
 			}
 		}
 		// The image copy
-		imagecopy($dst_im, $src_im, $dst_x, $dst_y, $src_x, $src_y, $src_w, $src_h);
+		imagecopy( $dst_im, $src_im, $dst_x, $dst_y, $src_x, $src_y, $src_w, $src_h );
 	}
 
-	private static function merge_images($dest, $src) {
-		self::imagecopymerge_alpha($dest, $src, 0, 0, 0, 0, 128, 128, 100);
+	private static function merge_images( $dest, $src ) {
+		self::imagecopymerge_alpha( $dest, $src, 0, 0, 0, 0, 128, 128, 100 );
 	}
 
-	private static function merge_alphamask($dest, $mask1, $mask2) {
-		imagesavealpha($dest, true);
-		imagealphablending($dest, false);
+	private static function merge_alphamask( $dest, $mask1, $mask2 ) {
+		imagesavealpha( $dest, true );
+		imagealphablending( $dest, false );
 
-		$w = imagesx($dest);
-		$h = imagesy($dest);
+		$w = imagesx( $dest );
+		$h = imagesy( $dest );
 
 		//loop through mask pixels and modify alpha for each pixel of dest image
-		for( $x = 0; $x < $w; $x++ ){
-			for( $y = 0; $y < $h; $y++ ){
+		for( $x = 0; $x < $w; $x++ ) {
+			for( $y = 0; $y < $h; $y++ ) {
 				// get alpha value from both masks
 				$colorxy = imagecolorat( $mask1, $x, $y );
 				$alpha1 = ( $colorxy >> 24 ) & 0xFF;
 
 				$colorxy = imagecolorat( $mask2, $x, $y );
 				$alpha2 = ( $colorxy >> 24 ) & 0xFF;
 
-				$alpha = min($alpha1, $alpha2);
+				$alpha = min( $alpha1, $alpha2 );
 
 				// get color from dest image
-				$colorxy = imagecolorat($dest, $x, $y);
+				$colorxy = imagecolorat( $dest, $x, $y );
 
 				//get the color index with new alpha
 				$alphacolorxy = imagecolorallocatealpha( $dest, ( $colorxy >> 16 ) & 0xFF, ( $colorxy >> 8 ) & 0xFF, $colorxy & 0xFF, $alpha );
 				//set pixel with the new color + opacity
-				imagesetpixel($dest, $x, $y, $alphacolorxy);
+				imagesetpixel( $dest, $x, $y, $alphacolorxy );
 			}
 		}
 	}
 
-	public static function uploadBadge($destinationFileName, $badgeLevel) {
+	public static function uploadBadge( $destinationFileName, $badgeLevel ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce the number of returns of this function 4, down to the maximum allowed 3. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1142)